Sources/Navigation/DistributedNavigationDelegate.swift

```diff
@@ -111,7 +111,8 @@ private extension DistributedNavigationDelegate {
 #endif
 
     /// continues until first non-nil Navigation Responder decision and returned to the `completion` callback
-    func makeAsyncDecision<T>(boundToLifetimeOf webView: WKWebView,
+    func makeAsyncDecision<T>(for actionDebugInfo: some CustomDebugStringConvertible,
+                              boundToLifetimeOf webView: WKWebView,
```

Comment:
Didn't we agree to disable this kind of warning?

Reply:
As I remember we agreed to make it less intrusive (to stop debugger and not cause assertion) and provide more info. This won't affect CI runners as there's no debugger but may help detecting some issues